hamilton/__init__.py

```diff
@@ -0,0 +1,6 @@
+try:

```

Comment:
Should we output this as `__version__`? Or keep it in caps...

Looking at this (old but I think still relevant, updated recently), I think we want the `__version__`. Can be tuple or string 🤷. https://stackoverflow.com/questions/458550/standard-way-to-embed-version-into-python-package.